### ISSUE TITLE
Navigation screen is created for testing prototype

### ISSUE THREAD

**Issue by ChristineRoseSteiffer** [department-of-veterans-affairs/tmf-auth-exp-design-patterns#84]
The prototype needs a navigation page where users can easily start tasks and move about the prototype as needed in order to respond to our questions.

```[tasklist]
### Tasks
- [x] Designs are created to indicate a navigation page and how users will get back to that page from any subsequent pages in throughout the prototype
- [x] Prototype is updated with that navigation page design and users are easily able to navigate to that page from any other page within the prototype
```

**Comment by beckyphung** [comment]
During sprint planning 8/13: @Jones-Morgan can possibly support after completing convo guide

**Comment by adamwhitlock1** [comment]
Will be working on this next along with bugfixes, now that the pre-pilot is done

**Comment by adamwhitlock1** [comment]
Made good progress on this today. I created the navigation page and have the majority of the navigation tabs set up and working. Here are some WIP screenshots

Main nagivation / catch all page
![Image](https://github.com/user-attachments/assets/3220571f-6a7e-41f8-99d2-84e26b0470f8)


Navigation tabs at the top of page
![Image](https://github.com/user-attachments/assets/bcfa36db-f6fd-4e07-8677-11e06b60fb61)

Still have some cleanup and testing to do, but I pushed all current code up to the branch:
https://github.com/department-of-veterans-affairs/vets-website/tree/AuthPatterns/aw/84/task-navigation-page

Will continue on Monday with the goal of getting PR up by Tuesday

**Comment by adamwhitlock1** [comment]
Submitted PR for this work here:
https://github.com/department-of-veterans-affairs/vets-website/pull/31483

Once it is approved and deployed to staging / codespaces, I will get the changes reviewed by the team.

**Comment by adamwhitlock1** [comment]
This work has been merged into the main branch and is ready to be validated / closed.

The codespace at https://musical-doodle-7676579pxr3rxrg-3001.app.github.dev/mock-form-ae-design-patterns can be used to view this work.

![Image](https://github.com/user-attachments/assets/df326d38-69ad-476f-9f82-35b2ab20bcc3)
